### PR TITLE
Fix get data

### PR DESCRIPTION
**Pull request by sylvihuynh**
Fix comment in #115.  Now all answers are returned and the current page is set to where to user left off, and the user can go back and get all the previous scores assessments.

### CODE REVIEW

File: pages/assessment.tsx [hunk 1]
```diff
@@ -167,6 +165,16 @@ const Assessment: NextPage = () => {
     }
   };
 
+  const findAndSetCurrentPage = (assessments: AssessmentType[]) => {
```
Comment (amalieeh) [inline pages/assessment.tsx:168]: Can this be called in `handleSetMaxItems()` as well? 
Replacing line 158-164.
The function would then need to take in `value` which basically is maxItemsPerPage

Comment (amalieeh) [inline pages/assessment.tsx:168]: I dont think you necessarily need to take in assessments? Or maybe you need it for the initial run?

Comment (sylvihuynh) [reply pages/assessment.tsx:168]: If we want to use it in `handleSetMaxItems()` we have to take in assessments as the initial run uses  assessments/answers from `getAssessmentData()` while `handleSetMaxItems()` uses `assessments from the state`

**Review by amalieeh — APPROVED** [review]
Nice and sleek 👌  Good work! This setup makes more sense to me 🙌

**Pull request merged**